python/src/phylanx.cpp

```diff
@@ -11,6 +11,9 @@
 #include <bindings/binding_helpers.hpp>
 
 #include <pybind11/pybind11.h>
+#include <pybind11/iostream.h>
+#include <pybind11/operators.h>
+#include <pybind11/stl.h>
```

Comment:
Why are these #include's necessary for this file?

Reply:
Sorry, let me clean up the unrelated stuff.